TwitchPlaysAssembly/Src/Votes.cs

```diff
@@ -0,0 +1,145 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Timers;
+
+public static class Votes
+{
+	public static bool _going = false;
+	public static bool _global = false;
+	public static Dictionary<string, bool> Voters = new Dictionary<string, bool>();
+	public static TwitchBomb Bomb;
+
+	private static Timer Countdown = null;
+	private static string ActionType = "";
+
+	private static Dictionary<string, Action> actionDict = new Dictionary<string, Action>()
+	{
+		{"Detonation", () => Bomb.CauseExplosionByModuleCommand("Voting ended. Detonating bomb...", "Voted detonation")},
+		{"VS Mode Toggle", () => SendAndDo("Voting ended. Toggling VS Mode...", () => OtherModes.Toggle(TwitchPlaysMode.VS)) }
+	};
+	private static Dictionary<string, int> percentageDict = new Dictionary<string, int>()
+	{
+		{ "Detonation", TwitchPlaySettings.data.MinimumYesForDetonation },
+		{"VS Mode Toggle", TwitchPlaySettings.data.MinimumYesForVSMode }
+	};
+
+	private static void SendAndDo(string message, Action action)
+	{
+		IRCConnection.SendMessage(message);
+		action();
+		return;
+	}
+
+	static Votes()
+	{
+		ResetTimer();
+	}
+
+	#pragma warning disable IDE0049
+	public static void OnTick(Object source, ElapsedEventArgs e)
+	{
+		if (!_going && !_global) return;
+		Countdown.Enabled = false;
+		int yesVotes = 0;
+		int noVotes = 0;
+		foreach (KeyValuePair<string, bool> Pair in Voters)
+		{
+			if (Pair.Value)
+			{
+				yesVotes += 1;
+				continue;
+			}
+			noVotes += 1;
+		}
+		if (yesVotes >= (yesVotes+noVotes)*((percentageDict[ActionType]%101)/100))
```

Comment:
```suggestion
		int yesVotes = Voters.Count(pair => pair.Value);
		if (yesVotes >= Voters.Count * (percentageDict[ActionType] / 100f))
```
This bit can be simplified. Also using modulo is probably not the best idea since it would wrap down to a low number which is probably not what the user expected.